### PR TITLE
Add Block componenet and get topic page working end to end

### PR DESCRIPTION
**Pull request by shifucun**
Page config is stored in initial page load as an html data field. Then each block (tile) renders async with their own data fetching.

Things to add later:
- real css
- ranking tile
- bar tile
- better config for place type, child place type and similar place type; place based stat vars..

### CODE REVIEW

**Review by beets — APPROVED** [review]
nice!

File: static/js/topic_page/block.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import React from "react";
+
+import { randDomId } from "../shared/util";
+import { StatVarMetadata } from "../types/stat_var";
+import { LineTile } from "./line_tile";
+import { MapTile } from "./map_tile";
+import { RankingTile } from "./ranking_tile";
+import { Tile } from "./tile";
+
+export interface BlockPropType {
```
Comment (chejennifer) [inline static/js/topic_page/block.tsx:30]: from the proto, it looks like a block can also contain a description, probably want to include a field for that here in the BlockPropType

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/topic_page/block.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/topic_page/block.tsx:30]: Done

**Comment by shifucun** [comment]
Thanks!

**Pull request merged**